keep.tex

```diff
@@ -293,7 +293,7 @@ \subsection{Secure multi party computation} \label{sMPC}
 the dealer, who can assemble the final output, without any participant
 learning more than their initial secret share.
 
-Intuitively, sMPC works like this.
+Intuitively, sMPC works like this:
```

Comment:
:laughing: when I wrote this I expected it'd be edited out- even with the colon it's funny style for a more academic paper

Reply:
<_<